build-docs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+root_path="src" # this could be src/interfaces
+# generate docs in a temporary directory
+temp_folder="technical-docs"
+
+FOUNDRY_PROFILE=docs forge doc --out "$temp_folder"
+
+# edit generated summary not to have container pages
+# - [jobs](src/interfaces/jobs/README.md)
+# should become
+# - [jobs]()
+# TODO
```

Comment:
I believe in most cases container pages make sense, since they explain the concept (jobs in this case) and provide additional information, gotchas, etc. @dristpunk @0xGorilla what do you think?

Reply:
> I believe in most cases container pages make sense, since they explain the concept (jobs in this case) and provide additional information, gotchas, etc. @dristpunk @0xGorilla what do you think?

makes sense for me